File: HelloTimeF/HelloTimeF/ViewController.m
//
//  ViewController.m
//  HelloTime
//
//  Created by Japnam Singh on 3/6/15.
//  Copyright (c) 2015 Humber. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [self checkTime:self];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (void)checkTime:(id)sender
{
    NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
    [formatter setDateFormat:@"h:mm:ss a"];
    [self.timeLabel setText:[formatter stringFromDate:[NSDate date]]];
    [self performSelector:@selector(checkTime:) withObject:self afterDelay:1.0];
    NSLog(@"Japnam Singh-824196513");
}

@end

File: Map/Map/ViewController.swift
//
//  ViewController.swift
//  Map
//
//  Created by JAPNAM on 5/19/15.
//  Copyright (c) 2015 HC. All rights reserved.
//

import UIKit
import MapKit
import CoreLocation
class ViewController: UIViewController {

    
    @IBOutlet var textLabel: UILabel!
    @IBOutlet var Map: MKMapView!
   
    @IBOutlet var txtEmail: UILabel!
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        
        var location = CLLocationCoordinate2DMake(43.729404, -79.6056209)
        
        
        var span = MKCoordinateSpanMake(0.002, 0.002)
       var region = MKCoordinateRegion(center: location, span: span)
        Map.setRegion(region, animated: true)
        var annotation = MKPointAnnotation()
     annotation.coordinate = location
        annotation.title = "Humber College"
        annotation.subtitle = "North Campus"
        
        Map.addAnnotation(annotation)

    
    
    
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

